src/Umbraco.Web/PropertyEditors/ValueConverters/MediaPickerValueConverter.cs

```diff
@@ -53,26 +65,31 @@ private bool IsOnlyImagesDataType(PublishedDataType dataType)
             return config.OnlyImages;
         }
 
-        public override object ConvertSourceToIntermediate(IPublishedElement owner, PublishedPropertyType propertyType, object source, bool preview)
+        public override object ConvertSourceToIntermediate(IPublishedElement owner, PublishedPropertyType propertyType,
+            object source, bool preview)
         {
             if (source == null) return null;
 
             var nodeIds = source.ToString()
-                .Split(new[] { "," }, StringSplitOptions.RemoveEmptyEntries)
+                .Split(new[] {","}, StringSplitOptions.RemoveEmptyEntries)
```

Comment:
well, we've all used a space around curlies until now... so I know if I work on this file, I'll revert this without even thinking - and I think even R# would do it for me... so, are you sure about it?

Reply:
Just, did a file format in Rider. Maybe I had to tweak the settings. Right now it uses default settings that should be based on what is used most in the code base. Can tweak this.